docs/support/ghc-version-support.md

```diff
@@ -75,23 +75,59 @@ Major versions of GHC which are not supported by HLS on master are extremely unl
 
 ## GHC version deprecation policy
 
-### Major versions
+### Base policy
+
+This is the static part of the policy that can be checked by a machine.
+
+#### Major versions
 
 A major GHC version is a "legacy" version if it is 3 or more major versions behind the latest GHC version that is
```

Comment:
are we like switching to the “oldest popular version” as min(stackage_lts, ghcup_recommended). As in https://discourse.haskell.org/t/call-for-opinions-shortening-the-hls-support-window/9641/16?u=soulomoon

Reply:
Yes, we are! Didn't have time to add this section yet :)

Reply:
Updated!